robusta_krr/main.py

```diff
@@ -125,6 +127,8 @@ def {func_name}(
                     verbose=verbose,
                     quiet=quiet,
                     log_to_stderr=log_to_stderr,
+                    file_output=file_output,
```

Comment:
I think it will be better to do something like:
output: slack / file.
it will be much more readable in the code